s3-pit-restore

```diff
@@ -58,7 +58,8 @@ class TestS3PitRestore(unittest.TestCase):
         if args.dest_prefix:
             base_path = os.path.join(args.dest_prefix, base_path)
 
-        s3 = boto3.resource('s3')
+        s3 = boto3.resource('s3', endpoint_url=None or args.endpoint_url)
```

Comment:
You should squash the two commits into one, it's only one commit the one I would merge.

Anyway, I don't think this really works. Have you tested it?

Shouldn't it be something like
```
endpoint_url = (args.endpoint_url if args.endpoint_url or None)
```
?

Reply:
I have tested, yes, with and without argument.
It's probably clearer with parenthesis : 
`s3 = boto3.resource('s3', endpoint_url=(None or args.endpoint_url))`